deploy/digital-ocean/koa/.github/workflows/api-staging.yml

```diff
@@ -19,6 +19,20 @@ on:
         required: false
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [ 16.x ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test api using jest
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - run: npm install
+      - run: npm test
```

Comment:
let's make a separate workflow file for tests running. It should triggers 
`on: pull_request`.
Also, add a guide, how to to block pull requests if tests fail.

https://stackoverflow.com/a/58655352